gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/x509/X509Config.groovy

```diff
@@ -72,6 +79,17 @@ class X509Config {
     }
   }
 
+  @ConditionalOnBean(X509StandaloneAuthConfig)
+  @Bean
+  AuthenticationManager authenticationManagerBean() {
+    new AuthenticationManager() {
+      @Override
+      Authentication authenticate(Authentication authentication) throws AuthenticationException {
+        throw new UnsupportedOperationException("No authentication should be done with this AuthenticationManager");
```

Comment:
This is specific to X509-_only_ use case. They have SAML+X509, where the startup problem from https://github.com/spinnaker/fiat/issues/123 doesn't exist. 

You're right that `authenticate` is never called in this use case.